### DISCUSSION TITLE
Difference between Claude.ai and https://console.anthropic.com/?

### DISCUSSION THREAD

**Discussion by rowild** [cline/cline#407] [Feature Requests]
I purchased a pro membership on Claude.ai, but it seems there is no API key available. A link on Claude.ai opens the https://console.anthropic.com/ where my current account on Claude.ai is not recognized ("Evaluation membership"). The help forum on that page as well as the chat cannot tell me if Claude.ai does provide an API key that I could use with the console.

So I wonder if anybody of you guys here could help me understand what's the difference between claude.ai and console.anthropic.com? And should I do in order to use this extension with an API key in VS Code? Thank you!

(If this is the wrong place to ask, would you be so kind and point me to a proper place? Thanks!)

**Comment by saoudrizwan** [comment]
Hey there! Your Claude subscription is separate from an API key. I suggest selecting 'OpenRouter' in the API Provider dropdown and setting up an account there, they have very generous rate limits too.

**Reply by mtnsh** [reply]
@saoudrizwan - Thanks for the tip, I try to understand the difference between the two. (openrouter / anthropic direct key)

put billing aside, is there a "goto" solution? is one better than the other? (my experience is that they differ in limits, mainly)

does openrouter ai "auto" works or tried in the implementation? (https://openrouter.ai/openrouter/auto)
thx

**Reply by rowild** [reply]
I use OpenRouter with the Cline Extension in VSCode (where code suggestions (artifacts?) are put directly into a diff window. Also, Cline can open the Terminal and a browser to check its own work). OpenRouter is indeed - as mentioned above - way more generous than Anthropic, and also a lot more stable, meaning that when Claude analyses something and then suggests code as an artifact, it can handle really long inputs; doing that with an anthropic key always - and I do mean: ALWAYS! - leads to code truncation or some other kind of interruption.

If you want to use the online Claude.ai you always need to copy all kind of code there, e.g. a project's config file and then the files you want Claude's help. It works, but it is more time consuming. So i personally use the Claude.ai for non-code related things (text optimisations, brain storming, etc).

**Comment by rowild** [comment]
Thank you very much, @saoudrizwan ! Much appreciated!
